CONTRIBUTING.md

```diff
@@ -44,14 +44,28 @@ $ npm run test:coverage
 
 ## Releasing
 
-This repository uses [lerna](https://lernajs.io) to manage and release the packages. Lerna automatically releases all the outdated packages. To check which packages are outdated and will be released, type `npx lerna updated`.
+This repository uses [lerna](https://lernajs.io) to manage and release the packages. Lerna automatically releases all the outdated packages. To check which packages are outdated and will be released, type `npm run publish:check`.
+
+If you have the ability to publish packages, you _must_ have [2FA enabled](https://docs.npmjs.com/getting-started/using-two-factor-authentication) on your npmjs.com account.
+
+### Before releasing
+
+Confirm that you're logged into npm, by running `npm whoami`. If you're not logged in, run `npm adduser` to login.
+
+If you're publishing a new package, ensure that its `package.json` file contains the correct `publishConfig` settings:
+
+```json
+	"publishConfig": {
+		"access": "public"
+	}
+```
 
 ### Development release
 
-Run the following command to release a dev version of the outdated packages:
+Run the following command to release a dev version of the outdated packages, replacing "123456" with your 2FA code:
```

Comment:
It may be worth making a note that since publishing can take some time, it is good to ensure that the one-time code still has a decent duration remaining before starting the process. Made more cumbersome by the fact that the developer needs to navigate through the publishing options before publishing starts.

Reply:
👍🏻